diff --git a/node_modules/flow-to-ts/flow-to-typescript.js b/node_modules/flow-to-ts/flow-to-typescript.js
index 04d034f..99371df 100644
--- a/node_modules/flow-to-ts/flow-to-typescript.js
+++ b/node_modules/flow-to-ts/flow-to-typescript.js
@@ -119061,6 +119061,8 @@ __webpack_require__.r(__webpack_exports__);
 /* harmony import */ var _src_convert_js__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_src_convert_js__WEBPACK_IMPORTED_MODULE_0__);
 
 console.log(_src_convert_js__WEBPACK_IMPORTED_MODULE_0___default()("\ntype A = {a: string}\n"));
+
+window.flow2ts = _src_convert_js__WEBPACK_IMPORTED_MODULE_0___default();
 /* harmony default export */ __webpack_exports__["default"] = (_src_convert_js__WEBPACK_IMPORTED_MODULE_0___default.a);
 
 /***/ }),
