diff --git a/node_modules/json_typegen_wasm/json_typegen_wasm.js b/node_modules/json_typegen_wasm/json_typegen_wasm.js
index 8baafc9..fe1d56d 100644
--- a/node_modules/json_typegen_wasm/json_typegen_wasm.js
+++ b/node_modules/json_typegen_wasm/json_typegen_wasm.js
@@ -1,5 +1,16 @@
 import * as wasm from './json_typegen_wasm_bg';
 
+if (IN_BROWSER) {
+    if (window.TextEncoder) {
+        return Promise.resolve();
+      } else {
+        return import("text-encoding-utf-8").then(encoding => {
+            window.TextEncoder = encoding.TextEncoder;
+            window.TextDecoder = encoding.TextDecoder;
+          });
+      }
+  }
+
 let cachedTextEncoder = new TextEncoder('utf-8');
 
 let cachegetUint8Memory = null;
