diff --git a/node_modules/json_typegen_wasm/json_typegen_wasm.js b/node_modules/json_typegen_wasm/json_typegen_wasm.js
index 8baafc9..3c9dc39 100644
--- a/node_modules/json_typegen_wasm/json_typegen_wasm.js
+++ b/node_modules/json_typegen_wasm/json_typegen_wasm.js
@@ -1,5 +1,11 @@
 import * as wasm from './json_typegen_wasm_bg';
 
+if (IN_BROWSER && !window.TextEncoder) {
+  const encoding = require("text-encoding-utf-8");
+  window.TextEncoder = encoding.TextEncoder;
+  window.TextDecoder = encoding.TextDecoder;
+}
+
 let cachedTextEncoder = new TextEncoder('utf-8');
 
 let cachegetUint8Memory = null;
