diff --git a/node_modules/next/dist/build/webpack-config.js b/node_modules/next/dist/build/webpack-config.js
index e856363..443b159 100644
--- a/node_modules/next/dist/build/webpack-config.js
+++ b/node_modules/next/dist/build/webpack-config.js
@@ -70,7 +70,8 @@ function getBaseWebpackConfig(dir, { dev = false, debug = false, isServer = fals
         sourceMap: false,
         cache: true,
         cpus: config.experimental.cpus,
-        distDir: distDir
+        distDir: distDir,
+      exclude: /\/babel-plugin-flow-to-typescript/
     };
     let webpackConfig = {
         mode: webpackMode,
